docs/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,288 @@
+    +------------|------------+                 +------------|------------+                     |                                                        
+                 |                                           |                                  |                                                        
+                 |                                           | YES                              |                                                        
+                 |                          +----------------|----------------+                 |                                                        
+   +--------------------------+             | Add comment                     |                 |                                                        
+   |   Add area label         |             | Remove needs-triage label       ------------------|                                                        
+   |   label: area/*          |             | label: triage/works-as-intended |                                                                          
+   +-------------|------------+             +---------------------------------+                                                                          
+                 |                                                                                                                                       
+                 |                        +----------+                                                                                                   
+                 |                        |   Done   ----------------------------------------                                                            
+                 |                        +----|-----+                                      |                                                            
+                 |                             |NO                                          |                                                            
+                 |                             |                         +------------------|------------------+                                         
+    +------------|-------------+          +----|----------------+ YES    |  Add details to issue               |                                         
+    |                          ------------  Signal Community?  ----------  label: help wanted                 |                                         
+    |Remove needs-triage label |          |                     |        |  label: beginner friendly (optional)|                                         
+    +--------------------------+          +---------------------+        +-------------------------------------+                                          
+                                                                                                                                                                                       
+``` 
```

Comment:
Shouldn't there be a workflow title line here, similar to the one for "Bug" type issues?

Reply:
Updated as suggested